examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/DeprecatedPackageCardAdapter.kt

```diff
@@ -38,12 +39,13 @@ class DeprecatedPackageCardAdapter(
             binding.currentPackage = currentPackage
             binding.isSubscription = product.type == ProductType.SUBS
             binding.isActive = activeSubscriptions.contains(product.id)
+            // Upgrades are no longer possible with deprecated methods.
```

Comment:
Honestly, I think it might make sense to just remove these deprecated classes now... I believe it was mostly used to test during the migration to the new ReplacementMode, but I think it wouldn't be very useful now... But this can also happen separately.

Reply:
Alright! My vote would be to do it separately indeed, to avoid this PR from creeping up in scope.

Reply:
Also quicker v8 beta 🚀 😄